### DISCUSSION TITLE
RequestSignature számítása fájlfeltöltés esetén

### DISCUSSION THREAD

**Discussion by tothj74** [nav-gov-hu/eVAT#59] [Kérdések és válaszok - Gépi interfész] [answered]
Az API Gateway Interface specifikáció 2.4.1 Számítás fájlfeltöltés esetén részében szereplő példa szerint:

A teljes requestSignature alapja így:
TSTKFT122256420171230182545ce-8f5e215119fa7dd621DLMRHRLH2S797EB337CB3FD673976F67DE36230DFEEB3A7BC62F68423DEB3607BB2
11EED7E57E8515A5B8C865B97799E16961EE83FE13D5A82A4951ADF4BB42C779832883B
A requestSignature értéke SHA3-512 hashelést és nagybetűsítést követően az alábbi lesz:
● BBC670463D11CFE8428F492807CA9086243B13015DA41605E077830EC37459543DE1C0965C2B
D1A9D8811FAFAED0D465107A93D8EA0E9BBC2ECB8DCA18FB2F17

A hivatkozott RequestSignature alapra a https://codebeautify.org/sha3-512-hash-generatort futtatva egész más értéket kapok. 
Van esetleg elírás a példában, esetleg az algoritmus leírásában?

**Comment by PeterAdam** [comment, marked as answer]
Kiszedve a pdf-ből másoláskor keletkezett sortöréseket, berakva egy elválasztójelet (ce-8f5e-215119fa7dd621DLMRHRLH2S) a dokumentáció szerinti kivonat áll elő nekem a fenti oldalon is

**Reply by tothj74** [reply]
Köszönöm
